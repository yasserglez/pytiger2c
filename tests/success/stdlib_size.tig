let
    var letter0 := ""
    var letter1 := "a"
    var letter2 := "ab"
    var letter3 := "abc"
    var letter4 := "abcd"
    var letter5 := "abcde"
    var letter6 := "abcdef"
    var letter7 := "abcdefg"
    var letter8 := "abcdefgh"
    var letter9 := "abcdefghi"
    var letter10 := "abcdefghij"
    var letter11 := "abcdefghijk"
    var letter12 := "abcdefghijkl"
    var letter13 := "abcdefghijklm"
    var letter14 := "abcdefghijklmn"
    var letter15 := "abcdefghijklmno"
    var letter16 := "abcdefghijklmnop"
    var letter17 := "abcdefghijklmnopq"
    var letter18 := "abcdefghijklmnopqr"
    var letter19 := "abcdefghijklmnopqrs"
    var letter20 := "abcdefghijklmnopqrst"
    var letter21 := "abcdefghijklmnopqrstu"
    var letter22 := "abcdefghijklmnopqrstuv"
    var letter23 := "abcdefghijklmnopqrstuvw"
    var letter24 := "abcdefghijklmnopqrstuvwx"
    var letter25 := "abcdefghijklmnopqrstuvwxy"
    var letter26 := "abcdefghijklmnopqrstuvwxyz"
    var letter27 := "abcdefghijklmnopqrstuvwxyzA"
    var letter28 := "abcdefghijklmnopqrstuvwxyzAB"
    var letter29 := "abcdefghijklmnopqrstuvwxyzABC"
    var letter30 := "abcdefghijklmnopqrstuvwxyzABCD"
    var letter31 := "abcdefghijklmnopqrstuvwxyzABCDE"
    var letter32 := "abcdefghijklmnopqrstuvwxyzABCDEF"
    var letter33 := "abcdefghijklmnopqrstuvwxyzABCDEFG"
    var letter34 := "abcdefghijklmnopqrstuvwxyzABCDEFGH"
    var letter35 := "abcdefghijklmnopqrstuvwxyzABCDEFGHI"
    var letter36 := "abcdefghijklmnopqrstuvwxyzABCDEFGHIJ"
    var letter37 := "abcdefghijklmnopqrstuvwxyzABCDEFGHIJK"
    var letter38 := "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKL"
    var letter39 := "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLM"
    var letter40 := "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMN"
    var letter41 := "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNO"
    var letter42 := "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOP"
    var letter43 := "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQ"
    var letter44 := "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQR"
    var letter45 := "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRS"
    var letter46 := "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRST"
    var letter47 := "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTU"
    var letter48 := "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUV"
    var letter49 := "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVW"
    var letter50 := "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWX"
    var letter51 := "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXY"
in
(
   (printi(size(letter0)); print(":"); print(letter0); print("\n"));
   (printi(size(letter1)); print(":"); print(letter1); print("\n"));
   (printi(size(letter2)); print(":"); print(letter2); print("\n"));
   (printi(size(letter3)); print(":"); print(letter3); print("\n"));
   (printi(size(letter4)); print(":"); print(letter4); print("\n"));
   (printi(size(letter5)); print(":"); print(letter5); print("\n"));
   (printi(size(letter6)); print(":"); print(letter6); print("\n"));
   (printi(size(letter7)); print(":"); print(letter7); print("\n"));
   (printi(size(letter8)); print(":"); print(letter8); print("\n"));
   (printi(size(letter9)); print(":"); print(letter9); print("\n"));
   (printi(size(letter10)); print(":"); print(letter10); print("\n"));
   (printi(size(letter11)); print(":"); print(letter11); print("\n"));
   (printi(size(letter12)); print(":"); print(letter12); print("\n"));
   (printi(size(letter13)); print(":"); print(letter13); print("\n"));
   (printi(size(letter14)); print(":"); print(letter14); print("\n"));
   (printi(size(letter15)); print(":"); print(letter15); print("\n"));
   (printi(size(letter16)); print(":"); print(letter16); print("\n"));
   (printi(size(letter17)); print(":"); print(letter17); print("\n"));
   (printi(size(letter18)); print(":"); print(letter18); print("\n"));
   (printi(size(letter19)); print(":"); print(letter19); print("\n"));
   (printi(size(letter20)); print(":"); print(letter20); print("\n"));
   (printi(size(letter21)); print(":"); print(letter21); print("\n"));
   (printi(size(letter22)); print(":"); print(letter22); print("\n"));
   (printi(size(letter23)); print(":"); print(letter23); print("\n"));
   (printi(size(letter24)); print(":"); print(letter24); print("\n"));
   (printi(size(letter25)); print(":"); print(letter25); print("\n"));
   (printi(size(letter26)); print(":"); print(letter26); print("\n"));
   (printi(size(letter27)); print(":"); print(letter27); print("\n"));
   (printi(size(letter28)); print(":"); print(letter28); print("\n"));
   (printi(size(letter29)); print(":"); print(letter29); print("\n"));
   (printi(size(letter30)); print(":"); print(letter30); print("\n"));
   (printi(size(letter31)); print(":"); print(letter31); print("\n"));
   (printi(size(letter32)); print(":"); print(letter32); print("\n"));
   (printi(size(letter33)); print(":"); print(letter33); print("\n"));
   (printi(size(letter34)); print(":"); print(letter34); print("\n"));
   (printi(size(letter35)); print(":"); print(letter35); print("\n"));
   (printi(size(letter36)); print(":"); print(letter36); print("\n"));
   (printi(size(letter37)); print(":"); print(letter37); print("\n"));
   (printi(size(letter38)); print(":"); print(letter38); print("\n"));
   (printi(size(letter39)); print(":"); print(letter39); print("\n"));
   (printi(size(letter40)); print(":"); print(letter40); print("\n"));
   (printi(size(letter41)); print(":"); print(letter41); print("\n"));
   (printi(size(letter42)); print(":"); print(letter42); print("\n"));
   (printi(size(letter43)); print(":"); print(letter43); print("\n"));
   (printi(size(letter44)); print(":"); print(letter44); print("\n"));
   (printi(size(letter45)); print(":"); print(letter45); print("\n"));
   (printi(size(letter46)); print(":"); print(letter46); print("\n"));
   (printi(size(letter47)); print(":"); print(letter47); print("\n"));
   (printi(size(letter48)); print(":"); print(letter48); print("\n"));
   (printi(size(letter49)); print(":"); print(letter49); print("\n"));
   (printi(size(letter50)); print(":"); print(letter50); print("\n"));
   (printi(size(letter51)); print(":"); print(letter51); print("\n"))
)
end
