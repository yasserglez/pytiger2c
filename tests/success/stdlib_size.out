0:
1:a
2:ab
3:abc
4:abcd
5:abcde
6:abcdef
7:abcdefg
8:abcdefgh
9:abcdefghi
10:abcdefghij
11:abcdefghijk
12:abcdefghijkl
13:abcdefghijklm
14:abcdefghijklmn
15:abcdefghijklmno
16:abcdefghijklmnop
17:abcdefghijklmnopq
18:abcdefghijklmnopqr
19:abcdefghijklmnopqrs
20:abcdefghijklmnopqrst
21:abcdefghijklmnopqrstu
22:abcdefghijklmnopqrstuv
23:abcdefghijklmnopqrstuvw
24:abcdefghijklmnopqrstuvwx
25:abcdefghijklmnopqrstuvwxy
26:abcdefghijklmnopqrstuvwxyz
27:abcdefghijklmnopqrstuvwxyzA
28:abcdefghijklmnopqrstuvwxyzAB
29:abcdefghijklmnopqrstuvwxyzABC
30:abcdefghijklmnopqrstuvwxyzABCD
31:abcdefghijklmnopqrstuvwxyzABCDE
32:abcdefghijklmnopqrstuvwxyzABCDEF
33:abcdefghijklmnopqrstuvwxyzABCDEFG
34:abcdefghijklmnopqrstuvwxyzABCDEFGH
35:abcdefghijklmnopqrstuvwxyzABCDEFGHI
36:abcdefghijklmnopqrstuvwxyzABCDEFGHIJ
37:abcdefghijklmnopqrstuvwxyzABCDEFGHIJK
38:abcdefghijklmnopqrstuvwxyzABCDEFGHIJKL
39:abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLM
40:abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMN
41:abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNO
42:abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOP
43:abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQ
44:abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQR
45:abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRS
46:abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRST
47:abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTU
48:abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUV
49:abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVW
50:abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWX
51:abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXY
